code/config.py

```diff
@@ -107,12 +107,12 @@ def scope_paths_and_parameters(paths, param):
     # Paths to the shapefiles
     PathTemp = root + "02 Shapefiles for regions" + fs + "User-defined" + fs
 
-    paths["spatial_scope"] = PathTemp + "gadm36_GHA_0.shp"
-    paths["subregions"] = PathTemp + "gadm36_GHA_0.shp"
+    paths["spatial_scope"] = PathTemp + "gadm36_ECU_0.shp"
+    paths["subregions"] = PathTemp + "gadm36_ECU_0.shp"
 
     # Name tags for the scope and the subregions
-    param["region_name"] = "Ghana"  # Name tag of the spatial scope
-    param["subregions_name"] = "gadm36_GHA_0"  # Name tag of the subregions
+    param["region_name"] = "Ecuador"  # Name tag of the spatial scope
+    param["subregions_name"] = "gadm36_ECU_0"  # Name tag of the subregions
```

Comment:
I disapprove of the name tags for subregions. Please see how I named them in the folders inside "Ecuador"